src/index.js

```diff
@@ -243,4 +243,8 @@ Monologue.mixInto( Broker );
 
 var broker = new Broker();
 
-module.exports = broker;
+Object.defineProperty(exports, "__esModule", {
```

Comment:
it looks like you have some generated code from Babeljs included in this pull request. was that intentional?